roles/solrcloud/tasks/install.yml

```diff
@@ -1,54 +1,38 @@
 ---
-# installation instructions taken from: https://lucene.apache.org/solr/guide/7_2/taking-solr-to-production.html#taking-solr-to-production
-
-- name: solrcloud | create solr dirs
-  ansible.builtin.file:
-    path: '{{ item }}'
-    state: directory
-    owner: '{{ solr_user }}'
-    group: '{{ solr_group }}'
-    mode: 0755
-  loop:
-    - '{{ solr_log_dir }}'
-    - '{{ solr_data_dir }}'
-
-- name: solrcloud | Check for solr service
+- name: Solrcloud | Check for solr service
   ansible.builtin.command: service solr status
   ignore_errors: true
   changed_when: false
   register: service_solrcloud_status
   args:
-    warn: false  # The service module doesn't appear to have a status state
+    warn: false
 
-- name: solrcloud | Download SolrCloud binaries
+- name: Solrcloud | Download SolrCloud binaries
   ansible.builtin.get_url:
     url: "{{ solr_cloud_url }}"
     dest: '/tmp/{{ solr_cloud_package }}'
     timeout: 300
+    mode: "0644"
   register: solr_downloaded
-  when:
-    service_solrcloud_status is failed
-  retries: 3
-  delay: 10
   until: solr_downloaded is succeeded
   ignore_errors: true
```

Comment:
hm, this seems risky - what happens when the URL changes? I'd think we want to know if this step fails.